analyzers/tests/SonarAnalyzer.Test/Rules/CloudNative/AzureFunctionsLogFailuresTest.cs

```diff
@@ -67,25 +67,32 @@ public class AzureFunctionsLogFailuresTest
         [DataRow(false, "log.IsEnabled(LogLevel.Warning);")]
         public void AzureFunctionsLogFailures_VerifyLoggerCalls(bool isCompliant, string loggerInvocation)
         {
-            var code = @$"
-using Microsoft.Azure.WebJobs;
-using Microsoft.Extensions.Logging;
-using System;
+            var x = builder.AddSnippet($$"""
```

Comment:
Maybe a better variable naming?
```suggestion
            var verifier = builder.AddSnippet($$"""
```